src/services/contracts/acbuMinting.service.ts

```
@@ -316,6 +389,17 @@ export class MintingService {
 
       return ContractClient.fromScVal(result) as boolean;
     } catch (error) {
+      if ((params as any).txId) {
+        try {
+          await prisma.transaction.update({
+            where: { id: (params as any).txId },
+            data: { status: "FAILED" }
```

ESLint

[error] Insert `,`

(prettier/prettier)